src/saving/SaveManager.cs

```diff
@@ -0,0 +1,157 @@
+using System;
```

Comment:
Did you bring this entire file back from the dead?
All the needed stuff should be in SaveHelper.cs